CRM/Utils/Check.php

```diff
@@ -209,7 +212,7 @@ public static function checkAll($max = FALSE) {
 
     foreach ($messages as $key => $message) {
       $hush = self::checkHushSnooze($message);
-      $message->setVisible(!$hush);
+      $messages[$key]->setVisible(!$hush);
```

Comment:
I find it strange that this was broken whereas the api was not (e.g. `civicrm_api3('System', 'check')` was correctly returning `is_visible` for each result). How is that possible? Does the system check api not wrap this function?